### ISSUE TITLE
Classifier for paket.dependencies file - needed for code completion and syntax highlighting.

### ISSUE THREAD

**Issue by hmemcpy** [fsprojects/Paket.VisualStudio#4]
Also, icon.

**Comment by vasily-kirichenko** [comment]
Why 

``` csharp
NuGetV3.FindPackages(FSharpOption<Paket.Utils.Auth>.None, "http://nuget.org/api/v2", "NUn")
```

immediately returns `string[0]`? What am I doing wrong?
/cc @forki

**Comment by forki** [comment]
yeah somthing is fucky since 5 min. Either I broke something or the service went down.

**Comment by forki** [comment]
http://preview.nuget.org/ver3-preview/index.json <- seems they disabled it.

**Comment by forki** [comment]
(FYI: https://github.com/hmemcpy/Paket.VisualStudio/tree/paket-intellisense )

**Comment by forki** [comment]
Ok @maartenba gave me the new url. It should be fixed in Paket 1.3.16

**Comment by vasily-kirichenko** [comment]
Yeah, I'm on that branch.

**Comment by vasily-kirichenko** [comment]
I `paket update` and it now works:

![image](https://cloud.githubusercontent.com/assets/873919/7480620/73d9f85e-f372-11e4-830e-91970617004e.png)

I'm not sure that the results are what I expected. I mean I think it should suggest packages that are _started_ with `NUn`, not containing it somewhere.

**Comment by forki** [comment]
Yep they implemented fuzzy search. Maybe we need to filter that list

**Comment by vasily-kirichenko** [comment]
There's a bug - when you select a package, it's name replaces the whole `nuget NUn` (i.e. it erases `nuget` part).

**Comment by vasily-kirichenko** [comment]
So, I think it's easy to finish intellisense for `paket.dependencies` file.

**Comment by hmemcpy** [comment]
@vasily-kirichenko thanks, I know that bug, fixing now.
